tests/Frameworks/WordPress/Version_4_8/wp-content/plugins/datadog/datadog.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/*
+ * @wordpress-plugin
+ * Plugin Name: Datadog Sample
+ * Plugin URI: https://www.datadoghq.com/
+ * Description: Just here for testing
+ * Version: 0.0.0
+ * Author: Datadog
+ * Author URI: https://www.datadoghq.com/
+ * License: GPLv2 or later
+ * Text Domain: datadog-sample
+*/
+
+function datadog_parse_request($wp) {
+	if (!isset($wp->query_vars['name'])) {
+		return;
+	}
+	if ('simple' === $wp->query_vars['name']) {
+		echo "Simple text endpoint\n";
+        exit;
```

Comment:
Is that the idiomatic WordPress way to do this simple endpoint?

Reply:
It looks like there are a number of options for adding custom endpoints in WP. A quick Google search unearthed [this resource](https://torquemag.io/2017/07/four-strategies-adding-custom-endpoints-wordpress-site/). It appears we're doing it the "old school" way here. :)

The custom endpoints for [the REST API](https://developer.wordpress.org/rest-api/extending-the-rest-api/adding-custom-endpoints/) would be nice, but the endpoints are all prefixed with `/wp-json`, so it wouldn't work for our tests without an additional workaround. But I'm totally open to some other method of adding custom endpoints to test if you think that's a good route to take. :)

Also the `/simple` test endpoint is an actual custom WP page that is in the DB dump for the tests.

Reply:
Okay. We should at least fix the formatting issue here ;)

Reply:
Lol - nice catch. Fixed. I went ahead and added a comment above the `remove_filter()` line you pointed out as well. :)